README.md

```diff
@@ -1,7 +1,14 @@
-## Understanding dbt-common
+## dbt-common
 
 The shared common utilities for dbt-core and adapter implementations use
 
+### Releasing dbt-common
+To release a new version of dbt-common to pypi, you'll need to: 
+1. Bump the `version` in [dbt_common_/__about__.py](https://github.com/dbt-labs/dbt-common/blob/main/dbt_common/__about__.py)
+2. Run the [release workflow](https://github.com/dbt-labs/dbt-common/actions/workflows/release.yml) to test pypi and confirm a successful test release in: https://test.pypi.org/project/dbt-common/
+3. Run the [release workflow](https://github.com/dbt-labs/dbt-common/actions/workflows/release.yml) to prod pypi and confirm a successful release in: https://pypi.org/project/dbt-common/
+
+
```

Comment:
Could we add a line about updating the dbt-common release in dbt-core:core/setup.py?

Reply:
I'm updating it to just be <1.0.0 so we only need to update it if we're releasing a new major version or a pre-release.

Reply:
updated!